platformio.ini

```diff
@@ -394,6 +403,14 @@ build_flags     = -DDEMO=1
                   -DUSE_SSD1306=1
                   ${dev_heltec_wifi_v3.build_flags}
 
+; Heltec LoRa V3 board, which is an S3 with USB-C and LoRa
+[env:helteclorav3demo]
+extends         = dev_heltec_wifi_lora_v3
+build_flags     = -DDEMO=1
+                  -DENABLE_WIFI=1
+                  -DUSE_SSD1306=1
```

Comment:
I know you copied this from the `heltecv3demo` env, but would you agree this should really be a define in the `dev_heltec_wifi_*` sections? (As in, it's a device trait, not a project one.)

Reply:
I don't understand the code enough yet to give my opinion on were that should be defined. I did it this way because it's the same as all the other heltc env.

Reply:
Fair enough. I'll chew on this myself some more when I have a little more time.